app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+from __future__ import annotations
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        [self.learn_skill(skill) for skill in ["JavaScript", "HTML", "CSS"]]
```

Comment:
what about using `extend`?

Reply:
I thought we suppose to use class instance methods. but, yeah, I can change that